pkgs/by-name/da/dash-mpd-cli/package.nix

```diff
@@ -1,33 +1,63 @@
 {
   lib,
-  stdenv,
+  stdenvNoCC,
   rustPlatform,
   fetchFromGitHub,
+  makeWrapper,
   protobuf,
+  ffmpeg,
+  libxslt,
+  shaka-packager,
 }:
 
+let
+  # dash-mpd-cli looks for a binary named `shaka-packager`, while
+  # shaka-packager provides `packager`.
+  shaka-packager-wrapped = stdenvNoCC.mkDerivation {
+    name = "shaka-packager-wrapped";
+    phases = [ "installPhase" ];
+    installPhase = ''
+      mkdir -p $out/bin
+      ln -s ${lib.getExe shaka-packager} $out/bin/shaka-packager
+    '';
+  };
```

Comment:
rather than make a shaka wrapper, would it make sense to rename the exe? What is the binary named on other distros?

Reply:
I can't find any other distros that package shaka-packager. [The AUR package](https://aur.archlinux.org/packages/shaka-packager-bin) uses [the upstream binary name](https://github.com/shaka-project/shaka-packager/releases) ([which is just `packager`](https://shaka-project.github.io/shaka-packager/html/documentation.html#getting-shaka-packager)).

Reply:
We can also add the symlink directly to the package. But I don't have any objections to renaming it to shaka-packager either, I think it's more ergonomic that way anyway.